pepper-apis/gae-egress-proxy/fluentd-logging.conf

```diff
@@ -0,0 +1,37 @@
+<source>
+  @type tail
+  <parse>
+    @type regexp
+
+    # Parse entire line as `message` but also grab the timestamp (`time` is the default capture name).
+    # Stackdriver will treat `message` as the `textPayload` of the log entry.
+    expression /^(?<message>(?<time>[^\|]*)\| .*)$/
+
+    # Timestamp format follows Ruby syntax.
+    time_format %Y/%m/%d %T
+  </parse>
+  path /var/log/squid/cache.log
+  pos_file /var/lib/google-fluentd/pos/squid-cache.pos
+  read_from_head true
+
+  # This is the log name that shows up in Stackdriver.
+  tag squid-cache
+</source>
+
+<source>
+  @type tail
+  <parse>
+    @type regexp
+
+    # Parse entire line as `message` but also pull out individual fields.
+    # All of these will end up in the `jsonPayload` in Stackdriver log entry.
+    expression /^(?<message>(?<time>[0-9]+[\.0-9]*) +(?<elapsed>[0-9]+) (?<source_ip>[0-9\.]+) (?<action>[A-Z_]+)\/(?<status_code>[0-9]+) (?<size>[0-9]+) (?<method>[A-Z]+) (?<url>[^ ]+) (?<username>[^ ]+) (?<peer_status>[^ ]+)/(?<peer_ip>[^ ]+) (?<mime>[^ ]+))/
```

Comment:
This is the default log format, per the document linked to in the ticket. Here's a nice little tool to try out the regex matching: http://fluentular.herokuapp.com/.

Reply:
Very cool!  Is there a way to [encode fields](https://cloud.google.com/error-reporting/docs/formatting-error-messages) to trigger error reporting?

Reply:
Just so I understand: is the idea to add additional fields for Error Reporting based on some conditional logic (e.g. if status_code is 500)? I think this should be possible using fluentd's filters, but I'll need to consult the documentation.

However, I'm not too sure about this endeavor at this moment. I'm not sure what in the logs will constitute something worthy of Error Reporting. For example, we can go off of the status_code, but if status_code is 5xx then shouldn't the clients report the errors and so we'll pick up on that either from `pepper-backend` or `housekeeping`? Certainly, if we find something interesting in the logs, we can add integration for it!

Reply:
Sounds like something for later, after we get some road time with the logs more generally.  All good!